### ISSUE TITLE
GTNH client side option to disable Hardcore Darkness should be eliminated.

### ISSUE THREAD

**Issue by YaNKeeR** [GTNewHorizons/NewHorizons#2649]
#### Which modpack version are you using?
GTHN v2.0.2.7
#
#### If in multiplayer; On which server does this happen? 
Stone Legion - any for that matter
#
#### What did you try to do, and what did you expect to happen?
There is a client side config for RandomThings that can turn off the hardcore darkness on the client that gives them an unfair advantage when playing on a server that other players are on who suffer the darkness.
#
#### What happend instead? (Attach screenshots if needed)

#
#### What do you suggest instead/what changes do you propose?
If there is a way to remove the config option to disable it so EVERYONE suffers the night the same way, not some clown who wants to cheat and turns it off locally.

**Comment by noobyaran9** [comment]
I suggest adding this mod.
https://minecraft.curseforge.com/projects/hardcore-darkness

But this mod breaks shader which is also one of a way to disable random things hardcore darkness. 

This mods forced the config to be the same as in the server.

**Comment by bartimaeusnek** [comment]
tbh i dont see any reason why this is important, especially because you can get nightvision pretty easy...

**Comment by YaNKeeR** [comment]
Nightvision is far enough in that you don't get it on night one.  I think removing the config option would be perfect.

**Comment by WarlordWossman** [comment]
If you really want to you can still use shaders and I don't think we should remove the ability to use one.

**Comment by Dream-Master** [comment]
We don't remove it is part of the pack.

**Comment by TwentyOneZ** [comment]
I vote up to using hardcore darkness mod!
Disabling the dark nights effect on client is too easy the way it is now, giving unfair advantage to people who do it in multiplayer servers.

**Comment by Dream-Master** [comment]
the problem is i am sure cant use hardcore darkness mod and random things. Is the same mod author and hardcore darkness is a part of the code from random things. I cant proof it the source is close.

**Comment by jiamingc0824** [comment]
You can add hardcore darkness mod in if u disable the hardcore darkness from random things configs.

**Comment by Dream-Master** [comment]
and this will work ? @jiamingc0824 did you test it ?

**Comment by jiamingc0824** [comment]
>and this will work ?

Yes. just make sure u disable hardcore darkness from random things before u add that mod. Also rmb to put a notice about the mod will break shader.

**Comment by Kortako** [comment]
I don't like that it breaks shaders. At this point, I feel like you're going to way too much trouble to "ensure" people will keep using hardcore darkness. If people want to change the config, that's their problem. But is it really an "unfair advantage"? Do other people even have advantages over others in this pack? It's PvE in pretty much every way.  And also, keep in mind a lot of people don't even play on the public servers, so this isn't even an "issue" for them.

Basically, if you could find a way to do it without breaking other things, I'd be fine with it. But not like this. That doesn't sit well with me.

**Comment by TwentyOneZ** [comment]
> Basically, if you could find a way to do it without breaking other things, I'd be fine with it. 

Hardcore Darkness doesn't stop people from using shaders. I tested it here and it worked fine with ShadersMod v2.3.31 for mc1.7.10, loaded a shader just fine. It says it is 'incompatible' because it doesn't get as dark as the mod's pitch-black nights would get with the shaders - with the shaders you can still see something in the dark, even though it is so much darker than using just the shader.

**You are still able to shaders, so it is not breaking anything.**

**Comment by TwentyOneZ** [comment]
> "And also, keep in mind a lot of people don't even play on the public servers"
Thinking about that, single players and custom-server people can even activate the 'moon light phase night lighting' feature that this mod have in single players, which adds a nice balance for those who doesn't like everyday pitch black nights, but also doesn't want it to be that much easier all the time. 
Its a win-win! =)

**Comment by richardhendricks** [comment]
Frankly it seems bogus anyways, someone can just hit the F7 key and see where the ground is located using NEI.

**Comment by Dream-Master** [comment]
i will close this issue and change nothing. All is fine as it is.
